Bottom: 7e9324f4bc6bed6e487a8af4f7f36cb559ab4572
Top:    d72d18d7f4d5c7dc8876f58182979137f1ec9894
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-07 21:53:17 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index be2487f..83cf471 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -456,7 +456,7 @@ int accept(int socket, struct sockaddr *addr, socklen_t *addrlen)
 			return ret;
 		}
 
-		fprintf(fout, "%d accept fd %d new fd\n", syscall(SYS_gettid), socket, index);
+		fprintf(fout, "%d accept fd %d new fd %d\n", syscall(SYS_gettid), socket, index);
 		fflush(fout);
 		fd_store(index, ret, type);
 		return index;
